src/views/AcademicCheckIn/components/PrivacyAgreement/privacy.json

```diff
@@ -0,0 +1,36 @@
+{
+  "FERPA":{
+      "header": "Notification of Rights Under FERPA",
```

Comment:
What drove the decision to put this content in a json file instead of inline in the JSX? Usually, we store in josn files data that has a repetitive structure (and so can be looped over with minimal JSX code compared to explicitly repeating it all), and data that changes frequently based on site-external factors. This doesn't seem like it fits either case, so I would think it is simpler and more understandable to put it in the JSX directly.

Reply:
It was primarily to save lines in the main file to improve readability as having long paragraphs that took up dozens of lines made it exhaustive to find specific code. Also this way if someone wants to update the information displayed it's much easier to find the paragraph they want to modify.